### PR TITLE
Skip webhook on kube-system by default for tap-injector/jaeger-injector

### LINKED ISSUE #11647: Don't trigger any webhook on kube-system
Currently the proxy-injector skips the kube-system namespace by default in its MutatingWebhookConfiguration namespaceSelector. We should do the same for the tap-injector and the jaeger-injector, whose selectors are currently empty. Although innocuous, this is currently generating a warning, in the Google Cloud Console for example (see https://linkerd.slack.com/archives/C89RTCWJF/p1700493518265689)

### PR DESCRIPTION
**Pull request by mikutas**
Fixes #11647

### CODE REVIEW

File: jaeger/charts/linkerd-jaeger/values.yaml [hunk 1]
```diff
@@ -285,10 +285,11 @@ webhook:
...
   namespaceSelector:
-    #matchExpressions:
-    #- key: runlevel
-    #  operator: NotIn
-    #  values: ["0","1"]
+    matchExpressions:
+    - key: kubernetes.io/metadata.name
+      operator: NotIn
+      values:
+      - kube-system
```
Comment (alpeb) [inline jaeger/charts/linkerd-jaeger/values.yaml:292]: I think adding a doc comment just above `namespaceSelector`  (like in the `linkerd-control-plane` chart) will avoid `bin/helm-doc` generating undesired entries for each one of these entries.

**Review by alpeb — COMMENTED** [review]
Thanks @mikutas !

**Review by alpeb — APPROVED** [review]
LGTM :+1:

**Review by mateiidavid — APPROVED** [review]
Looks good! Thanks @mikutas!

**Pull request merged**